virttest/env_process.py

```diff
@@ -1407,6 +1407,10 @@ def postprocess(test, params, env):
     if err:
         raise RuntimeError("Failures occurred while postprocess:\n%s" % err)
 
+    if params.get("env_cleanup", "no") == "yes":
+        if env:
+            env.destroy()
```

Comment:
cleaning it here would override the framework currently being handled, so to avoid the conflict can we can have this condition in test.py,

https://github.com/avocado-framework/avocado-vt/blob/770c2a27f15da28016925d79af4380dfb1ce5a68/avocado_vt/test.py#L417-L434


```python
if self.__safe_env_save(env) or params.get("env_cleanup", "no") == "yes":
    env.destroy()   # Force-clean as it can't be stored
```

Reply:
Sure, I can validate that.

Reply:
That's a right place, Tnx. have modified it.